Wikipedia/Code/VanishAccountPopUpAlertView.swift

```diff
@@ -0,0 +1,134 @@
+        static let thirdItem = WMFLocalizedString("vanish-modal-item-3", value: "If you have further questions about vanishing please visit", comment: "Text indicating that more infor is in the following link")
+        static let linkTitle = WMFLocalizedString("vanishing-link-title", value: "Wikipedia:Courtesy vanishing.", comment: "Courtesy vanishing page title")
+    }
+    
+    var body: some View {
+        GeometryReader { geometry in
+            ZStack {
+                Color.black.opacity(isVisible ? 0.3 : 0).edgesIgnoringSafeArea(.all)
+                if isVisible {
+                    ScrollView(.vertical, showsIndicators: false) {
+                        Spacer()
+                            .frame(height: geometry.size.height / 5)
+                        VStack(alignment: .center, spacing: 0) {
+                            Text(LocalizedStrings.title)
+                                .frame(maxWidth: .infinity)
+                                .frame(height: 45, alignment: .center)
+                                .font(Font(titleFont))
+                                .foregroundColor(Color(theme.colors.primaryText))
+                                .padding(10)
+                            Image("vanish-account-two-tone")
+                                .resizable()
+                                .scaledToFill()
+                                .frame(width: 85, height: 85, alignment: .center)
+                            BulletListView(theme: theme)
+                                .background(Color(theme.colors.paperBackground))
+                                .padding([.top, .leading, .trailing], 20)
+                                .frame(maxWidth: .infinity, alignment: .leading)
+                            Divider()
+                            Button(action: {
+                                withAnimation(.linear(duration: 0.3)) {
+                                    userInput = ""
+                                    isVisible = false
+                                }
+                            }, label: {
+                                Text(CommonStrings.okTitle)
+                                    .frame(maxWidth: .infinity)
+                                    .frame(height: 43, alignment: .center)
+                                    .foregroundColor(Color(theme.colors.link))
+                                    .font(Font(titleFont))
+                            }).buttonStyle(PlainButtonStyle())
+                        }
+                        .frame(maxWidth: thisWidth)
+                        .background(Color(theme.colors.paperBackground))
+                        .cornerRadius(14)
+                    }
+                  
+                }
+            }.frame(minHeight: geometry.size.height)
+        }
+    }
+    
+    private func updateOrientation() {
```

Comment:
@mazevedofs I don't think you're using the orientation stuff anymore, I think you can remove this method and the orientation @State vars. Feel free to push back if I'm misreading.